source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -1308,10 +1308,11 @@ Error handling
 Network error during server check
 `````````````````````````````````
 
-When a server `check`_ fails due to a network error,
-the client SHOULD clear its connection pool for the server:
+When a server `check`_ fails due to a network error (including a timeout),
+the client MUST clear its connection pool for the server:
 if the monitor's socket is bad it is likely that all are.
-(See `JAVA-1252 <https://jira.mongodb.org/browse/JAVA-1252>`_.)
+(See `JAVA-1252 <https://jira.mongodb.org/browse/JAVA-1252>`_,
+`SPEC-1396 <https://jira.mongodb.org/browse/SPEC-1396>`_.)
```

Comment:
SPEC tickets are not public, I don't think we can include this in our public documentation

Reply:
+1

Reply:
Removed.